drivers/usb/device/Kconfig

```diff
@@ -152,11 +152,13 @@ choice USB_MCUX_CONTROLLER_TYPE
 config USB_DC_NXP_EHCI
 	bool "MXRT EHCI USB Device Controller"
 	select NOCACHE_MEMORY if HAS_MCUX_CACHE
+	select USB_DC_HAS_HS_SUPPORT
 	help
 	  Kinetis and RT EHCI USB Device Controller Driver.
 
 config USB_DC_NXP_LPCIP3511
 	bool "LPC USB Device Controller"
+	select USB_DC_HAS_HS_SUPPORT
```

Comment:
Are there no FS-only controller supported by this driver?

Reply:
Not yet, I plan to add support for the FS controller.